src/test/resources/dokumentasjon/features/hovedregler/regelflyt/tredjelandsborgere/regelflyt_regel_19_3_1.feature

```diff
@@ -45,9 +45,9 @@ Egenskap: Regelflyt for regel 19.3.1
     Så skal svaret være "<Medlemskap>"
     Og skal regel "19.3.1" gi svaret "<Regel 19.3.1>"
     Og skal regel-årsaker være "<Årsaker>"
-    
+
     Eksempler:
       | Opphold til og med | Arbeid fra og med | Arbeid til og med | Regel 19.3.1 | Medlemskap | Årsaker |
       | 20.03.2017         | 01.01.2017        |                   | Ja           | Ja         |         |
-      | 20.01.2017         | 01.01.2017        | 30.01.2017        | Nei          | Uavklart   | 19.3.1  |
-      | 20.01.2017         | 31.12.2016        | 15.01.2017        | Nei          | Uavklart   | 19.3.1  |
\ No newline at end of file
+      | 20.01.2017         | 01.01.2017        | 30.01.2017        | Nei          | Uavklart   | 19.7    |
```

Comment:
Det er vel ønskelig at årsaker fortsatt skal være 19.3.1 ?

Reply:
Hadde en diskusjon med Helle og Per Birger på dette, der vi kom frem til at det ikke var så farlig om årsaken viste regel 19.7, men jeg kan godt høre med Per Birger en gang til for å være sikker.

Regel 19.7 kommer til å bli slettet mot slutten av dette året også